### ISSUE TITLE
create documentation on rsyslog use in containers

### ISSUE THREAD

**Issue by rgerhards** [rsyslog/rsyslog-doc#491]

**Comment by deoren** [comment]
Since this is an long-running ticket on the same topic, I'll post the suggestions I made as doc comments here.

These remarks are in regards to the `source/containers/container_features.rst` file.

> Upon start, rsyslog checks if it is running as pid 1. If so, this is a clear indication it is running inside a container.

- Explain why this is a clear indication. Perhaps make a quick reference to what normally holds pid 1 for anyone new to UNIX process management

> This changes some parameter defaults

- Add details explaining WHY the parameter defaults were changed.

refs #494

**Comment by deoren** [comment]
TODO: Reference official Docker Hub (page, profile, id?): 

https://hub.docker.com/u/rsyslog/

**Comment by rgerhards** [comment]
I'll add it when I finally modify the PR ;-)

**Comment by deoren** [comment]
> @rgerhards: I'll add it when I finally modify the PR ;-)

Yep. Is it helpful to add standalone/misc items like that to existing tickets (which appear to be works in progress) or as separate tickets? I was thinking the former based on the state.
